package.json

```diff
@@ -1,5 +1,15 @@
 {
+  "name": "tastytrails",
+  "version": "1.0.0",
+  "description": "**TastyTrails** is more than just recipes — it’s a vibrant social network where food lovers can connect, discover, and share the unique flavors of their cultures. A platform for passionate food enthusiasts to dive into a world of tastes!",
+  "main": "index.js",
+  "scripts": {
+    "start": "concurrently \"npm run dev --prefix backend\" \"npm run dev --prefix frontend\""
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
   "devDependencies": {
```

Comment:
@INam1995 did you write this or autogenerated? If you write then please remove

Reply:
done @AlfiyaSiddique

Reply:
> @INam1995 did you write this or autogenerated? If you write then please remove

@AlfiyaSiddique it's autogenerated